quill-core/src/main/scala/io/getquill/context/ActionMacro.scala

```diff
@@ -37,18 +37,25 @@ class ActionMacro(val c: MacroContext) extends ContextMacro with ReifyLiftings {
     translateBatchQueryPrettyPrint(quoted, q"false")
 
   def translateBatchQueryPrettyPrint(quoted: Tree, prettyPrint: Tree): Tree =
-    expandBatchAction(quoted) { case (batch, param, expanded) =>
-      q"""
+    expandBatchActionNew(quoted, false) {
```

Comment:
Can we use named variables when using boolean like this, please?
It's easier to read

Example
```scala
    expandBatchActionNew(quoted, isImportant = false) {
```